compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/viewinterop/UIKitInteropElementHolder.uikit.kt

```diff
@@ -147,4 +205,123 @@ internal abstract class UIKitInteropElementHolder<T : InteropView>(
     override fun changeInteropViewIndex(root: InteropViewGroup, index: Int) {
         root.insertSubview(view = group, atIndex = index.toLong())
     }
+
+    /**
+     * Check that [group] doesn't entirely clip a child view with a [cgRect]
+     */
+    private fun isCgRectEntirelyClippedByGroup(cgRect: CValue<CGRect>): Boolean =
```

Comment:
It seems semantic of this is simpler than name `isVisible` probably?

Reply:
Yes, refactored it.